webapp/channels/src/utils/url.tsx

```diff
@@ -102,11 +102,15 @@ export function makeUrlSafe(url: string, defaultUrl = ''): string {
 }
 
 export function getScheme(url: string): string | null {
-    const match = (/([a-z0-9+.-]+):/i).exec(url);
+    const match = (/^!?([a-z0-9+.-]+):/i).exec(url);
```

Comment:
I'm surprised that this regex didn't check for the start of the string before, but why are we allowing an exclamation mark here?

Reply:
Ah, I see. We should actually remove those URLS prefixed with `!` since we made that logic completely unnecessary a while back by making all external links open in a new tab by default